sklearn/semi_supervised/label_propagation.py

```diff
@@ -342,6 +343,22 @@ class LabelPropagation(BaseLabelPropagation):
     LabelSpreading : Alternate label propagation strategy more robust to noise
     """
 
+    def __init__(self, kernel='rbf', gamma=20, n_neighbors=7,
+                 alpha=None, max_iter=30, tol=1e-3, n_jobs=1):
+        # alpha is deprecated for LabelPropagation because it doesn't have any
+        # theoretical meaning (from the reference paper). Look at PR 6727.
+        if alpha is not None:
+            warnings.warn(
+                "Deprecated in 0.19 and it's going to be removed in 0.21.",
+                DeprecationWarning
+            )
+        alpha = .0
```

Comment:
It's a bit unfortunate but it looks like ``LabelPropagation`` needs to get it's own ``fit`` which calls
``super(LabelPropagation, self).fit``. But before, it checks whether alpha is set and raises a deprecation warning.
Then in ``BaseLabelPropagation`` you need to introduce a local variable ``alpha`` that has the right value. One way to do this is to check ``isinstance(self, LabelSpreading)`` - which is not great.
The other would be so set ``self._alpha = 0`` in ``LabelPropagation.fit``
and do ``alpha = getattr(self, "_alpha", self.alpha)`` in ``BaseLabelPropagation.fit``.